src/main/scala/nl/knaw/dans/easy/ingest/EasyIngest.scala

```diff
@@ -127,6 +126,7 @@ object EasyIngest {
     for {
       foxml <- getFOXML(sdo)
       pid <- executeIngest(configDictionary(sdo.getName), foxml)
+      _ = log.info(s"ingested $pid $foxml")
```

Comment:
`foxml` is a File object. I am not sure if the file name that will be printed here will be very `info`rmative, but maybe @jo-pol can argue it will.